logdetective/server/server.py

```diff
@@ -8,13 +8,30 @@
 
 import matplotlib
 import matplotlib.pyplot
-from fastapi import FastAPI, HTTPException, BackgroundTasks, Depends, Header, Request
+from fastapi import (
+    FastAPI,
+    HTTPException,
+    BackgroundTasks,
+    Depends,
+    Header,
+    Path,
+    Request,
+)
 
 from fastapi.responses import StreamingResponse
 from fastapi.responses import Response as BasicResponse
 import aiohttp
 import sentry_sdk
 
+from logdetective.server.exceptions import KojiInvalidTaskID
+
+from logdetective.server.database.models.koji import (
+    KojiTaskAnalysis,
+    KojiTaskAnalysisTimeoutError,
```

Comment:
I don't think these go against any guidelines, not that I care much about those, but for clarity it may be better to import them from module they are defined in.

Reply:
Yeah, those imports were generated by my IDE and I didn't check to make sure they were imported from the primary source. I'll adjust it.